seml/database.py

```diff
@@ -277,3 +281,40 @@ def clean_unreferenced_artifacts(db_collection_name=None, yes=False):
     logging.info('Deleting not referenced artifacts...')
     delete_files(db, not_referenced_artifacts, progress=True)
     logging.info(f'Successfully deleted {n_delete} not referenced artifact{s_if(n_delete)}.')
+
+def list_database(pattern, mongodb_config=None, progress=False, list_empty=False):
+    """
+    Prints a tabular version of multiple collections and their states (without resolving RUNNING experiments that may have been canceled manually).
+
+    Parameters
+    ----------
+    pattern : str
+        The regex collection names have to match against
+    mongodb_config : dict or None
+        A configuration for the mongodb. If None, the standard config is used.
+    progress : bool
+        Whether to use a progress bar for fetching
+    list_empty : bool
+        Whether to list collections that have no documents associated with any state
+    """
+    logging.warning(f"Status of {States.RUNNING[0]} experiments may not reflect if they have died or been canceled. Use `seml ... status` instead.")
+    if mongodb_config is None:
+        mongodb_config = get_mongodb_config()
+    db = get_database(**mongodb_config)
+    collection_names = [name for name in db.list_collection_names()
+                        if name not in ('fs.chunks', 'fs.files') and re.compile(pattern).match(name)]
+    name_to_counts = {}
```

Comment:
To remove the counter usage:
`name_to_counts = defaultdict(lambda: defaultdict(lambda: 0))`
and later:
`name_to_counts[collection_name].update({result['_id'] : result['_count'] for result in counts_by_status})`

Reply:
What's the issue with Counters? Isn't `Counter` specifically developed as a nice way to do `defaultdict(lambda: 0)`?

Reply:
That's an interesting take on counter. Never thought about it that way ^^ while functionally true but wording wise it seems a bit confusing as I at least first thought that something is counted there

Reply:
But we see each status only exactly once since the counting has been done by mongodb. So it's technically just a defaultdict?

Reply:
True, but still the (default)dict represents what you'd normally represent with a counter. I always use this data structure for dicts that are like that, mostly because of convenience functions (which we do not need here), consistency and readability. But I also don't wanna get lost in nitpicky stuff, we can also use defaultdicts.